data/events/2018-detroit.yml

```diff
@@ -33,8 +33,8 @@ nav_elements: # List of pages you want to show up in the navigation of your page
   - name: propose
```

Comment:
can comment this out as proposals aren't being accepted anymore.  Would also need to remove from the welcome page.

Reply:
done